lix/packages/sdk/src/combine/combine-changes.ts

```diff
@@ -0,0 +1,106 @@
+import type { LixPlugin } from "../plugin.js";
+import type { Change } from "../schema.js";
+import type { Lix } from "../types.js";
+
+/**
+ * Combines the changes of two lixes into the target lix.
+ */
+export async function combineChanges(args: { target: Lix; source: Lix }): Promise<void> {
+	const toBeAppliedChanges: Record<LixPlugin["key"], Array<Change>> = {};
```

Comment:
this needs to be in a transaction , otherwise the db will be in inconsistent state in case of errors

Reply:
implemented